cursorless-talon/src/spoken_forms.json

```diff
@@ -0,0 +1,263 @@
+      "funk name": "functionName",
+      "funk": "namedFunction",
+      "if state": "ifStatement",
+      "instance": "instance",
+      "item": "collectionItem",
+      "key": "collectionKey",
+      "lambda": "anonymousFunction",
+      "list": "list",
+      "map": "map",
+      "name": "name",
+      "regex": "regularExpression",
+      "section": "section",
+      "-one section": "sectionLevelOne",
+      "-two section": "sectionLevelTwo",
+      "-three section": "sectionLevelThree",
+      "-four section": "sectionLevelFour",
+      "-five section": "sectionLevelFive",
+      "-six section": "sectionLevelSix",
+      "selector": "selector",
+      "state": "statement",
+      "branch": "branch",
+      "type": "type",
+      "value": "value",
+      "condition": "condition",
+      "unit": "unit",
+      "element": "xmlElement",
+      "tags": "xmlBothTags",
+      "start tag": "xmlStartTag",
+      "end tag": "xmlEndTag",
+      "part": "part",
+      "chapter": "chapter",
+      "subsection": "subSection",
+      "subsubsection": "subSubSection",
+      "paragraph": "namedParagraph",
+      "subparagraph": "subParagraph",
+      "environment": "environment",
+      "command": "command",
+      "char": "character",
+      "word": "word",
+      "token": "token",
+      "identifier": "identifier",
+      "line": "line",
+      "sentence": "sentence",
+      "block": "paragraph",
+      "file": "document",
+      "paint": "nonWhitespaceSequence",
+      "short paint": "boundedNonWhitespaceSequence",
+      "link": "url",
+      "cell": "notebookCell"
+    },
+    "surrounding_pair_scope_type": {
+      "string": "string"
+    }
+  },
+  "specialMarks": {
+    "special_mark": {
+      "this": "currentSelection",
+      "that": "previousTarget",
+      "source": "previousSource",
+      "nothing": "nothing"
+    },
+    "unknown_symbol": { "special": "unknownSymbol" },
+    "line_direction": {
+      "row": "lineNumberModulo100",
+      "up": "lineNumberRelativeUp",
+      "down": "lineNumberRelativeDown"
+    }
+  },
+  "scopeVisualizer": {
+    "show_scope_visualizer": { "visualize": "showScopeVisualizer" },
+    "hide_scope_visualizer": { "visualize nothing": "hideScopeVisualizer" },
+    "visualization_type": {
+      "removal": "removal",
+      "iteration": "iteration"
+    }
+  },
+  "experimental.actions": {
+    "experimental_action": {
+      "-from": "experimental.setInstanceReference"
+    }
+  },
+  "wrapperSnippets": {
+    "wrapper_snippet": {
+      "else": "ifElseStatement.alternative",
+      "funk": "functionDeclaration.body",
+      "if else": "ifElseStatement.consequence",
+      "if": "ifStatement.consequence",
+      "try": "tryCatchStatement.body",
+      "link": "link.text"
+    }
+  },
+  "insertionSnippets": {
+    "insertion_snippet_no_phrase": {
+      "if": "ifStatement",
+      "if else": "ifElseStatement",
+      "try": "tryCatchStatement"
+    }
+  },
+  "insertionSnippetsSinglePhrase": {
```

Comment:
if these are just names of csvs, why not make that even clearer and just use the csv name as the key, eg `insertion_snippet_single_phrase.csv`, and even for the experimental ones just go eg `experimental/miscellaneous.csv`

Reply:
I would argue to keep it simple.  This should either cater completely to Python or not at all.  The in-between setup means it's neither intuitive coming from the TS or the Python side.  But let's discuss at meet up in next half hour